DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -2583,17 +2589,17 @@ internal async Task CreateInteractionResponseAsync(ulong interaction_id, string
             await this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.POST, route, payload: DiscordJson.SerializeObject(pld));
         }
 
-        internal async Task<DiscordMessage> GetOriginalInteractionResponseAsync(string interaction_token)
+        internal async Task<DiscordMessage> GetOriginalInteractionResponseAsync(ulong application_id, string interaction_token)
         {
-            var application_id = this.Discord.CurrentApplication.Id;
             var route = $"{Endpoints.WEBHOOKS}/:application_id/:interaction_token{Endpoints.MESSAGES}{Endpoints.ORIGINAL}";
             var bucket = this.Rest.GetBucket(RestRequestMethod.GET, route, new { application_id, interaction_token }, out var path);
 
             var url = Utilities.GetApiUriFor(path);
             var res = await this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.GET, route);
-            var json = JObject.Parse(res.Response);
+            var ret = JsonConvert.DeserializeObject<DiscordMessage>(res.Response);
 
-            return json.ToDiscordObject<DiscordMessage>();
```

Comment:
Why was this changed.

Reply:
Consistency with other methods

Reply:
Thanks for reminding me that I need to submit a PR regarding this

According to Maxine, `.ToDiscordObject` is the preferred method of deserialization